web-bundle/src/main/java/com/graphhopper/resources/IsochroneResource.java

```diff
@@ -117,15 +122,31 @@ public Response doGet(
                 JsonFeature feature = new JsonFeature();
                 HashMap<String, Object> properties = new HashMap<>();
                 properties.put("bucket", features.size());
+                if (respType.equalsIgnoreCase("geojson")) {
+                    // If you replace GraphHopper with your own brand name, this is fine.
+                    // Still it would be highly appreciated if you mention us in your about page!
+                    String[] copyrights = {"GraphHopper", "OpenStreetMap contributors"};
+                    properties.put("copyrights", copyrights);
```

Comment:
I have slightly updated master to reduce this to one line:

properties.put("copyrights", WebHelper.COPYRIGHTS);

Can you update to master and also remove the comment?

Reply:
Done